slick.core.js

```diff
@@ -675,10 +693,243 @@
   function regexSanitizer(dirtyHtml) {
      return dirtyHtml.replace(/(\b)(on[a-z]+)(\s*)=|javascript:([^>]*)[^>]*|(<\s*)(\/*)script([<>]*).*(<\s*)(\/*)script(>*)|(&lt;)(\/*)(script|script defer)(.*)(&gt;|&gt;">)/gi, '');
   }
+
+  // With help from https://youmightnotneedjquery.com/
+  function grep( elems, callback, invert ) {
+		var callbackInverse,
+			matches = [],
+			i = 0,
+			length = elems.length,
+			callbackExpect = !invert;
+
+		// Go through the array, only saving the items
+		// that pass the validator function
+		for ( ; i < length; i++ ) {
+			callbackInverse = !callback( elems[ i ], i );
+			if ( callbackInverse !== callbackExpect ) {
+				matches.push( elems[ i ] );
+			}
+		}
+
+		return matches;
+	}
+
+  function offset(el) {
+
+    box = el.getBoundingClientRect();
+    docElem = document.documentElement;
+    return {
+      top: box.top + window.pageYOffset - docElem.clientTop,
+      left: box.left + window.pageXOffset - docElem.clientLeft
+    };
  
-  // exports
-  $.extend(true, window, {
-    "Slick": {
+	}
+
+  function width(el, value)
+  {
```

Comment:
could you use the JS standard and move the opening curly brace on the same line of the `function ...() {`, that is a request for all other lines with same problem